README.md

```diff
@@ -43,6 +43,9 @@ A list of companies that have paid Developer Community Writer Programs.
 - [Code Magic](https://blog.codemagic.io/write-for-codemagic-ci-cd/) - Applications are currently closed.
   > Technical focused articles. Specific on Flutter.
 
+- [CodingSight](https://codingsight.com) - Proposed article should be sent to [marketing@devart.com](mailto:marketing@devart.com) (payment depends on the length, research and audience)
```

Comment:
do you have a range for this @HadiFadl ?

Reply:
@malgamves they removed the payment range from the website. It was published on [article levels](https://codingsight.com/article-levels/) and [Write for us](https://codingsight.com/write-for-us/). Unfortunately, These pages were removed recently, and I don't remember the range anymore. But, if you check the bottom of the [main page](https://codingsight.com/), you will see the "Write for US" announcement.

Reply:
@malgamves Currently, this is their only announcement: 

> CodingSight is open for new authors and partnership proposals. Send your offers and ideas to marketing@devart.com and we will get back to you at the earliest opportunity.

Reply:
@malgamves @Even for MSSQLTips, I don't know the range